pkgs/pinned.nix

```
@@ -1,13 +1,18 @@
 let
   nixpkgsPinned = import ./nixpkgs-pinned.nix;
-  unstable = import nixpkgsPinned.nixpkgs-unstable { config = {}; overlays = []; };
-  nixBitcoinPkgsUnstable = import ./. { pkgs = unstable; };
+  nixpkgsUnstable = import nixpkgsPinned.nixpkgs-unstable { config = {}; overlays = []; };
+  nixpkgsStable = import nixpkgsPinned.nixpkgs { config = {}; overlays = []; };
+  nixBitcoinPkgsUnstable = import ./. { pkgs = nixpkgsUnstable; };
+  nixBitcoinPkgsStable = import ./. { pkgs = nixpkgsStable; };
```

Nit: Put stable before unstable.
```suggestion
  nixpkgsStable = import nixpkgsPinned.nixpkgs { config = {}; overlays = []; };
  nixpkgsUnstable = import nixpkgsPinned.nixpkgs-unstable { config = {}; overlays = []; };
  nixBitcoinPkgsStable = import ./. { pkgs = nixpkgsStable; };
  nixBitcoinPkgsUnstable = import ./. { pkgs = nixpkgsUnstable; };
```